manifests/dev/vsphere-7.0u2/guestcluster/1.17/pvcsi.yaml

```diff
@@ -60,7 +60,7 @@ rules:
     verbs: ["use"]
     resourceNames: ["vmware-system-privileged"]
 ---
-kind: ClusterRole
+kind: Role
```

Comment:
Can you merge the these rbac rules in the `Role` above? That is for node too.

Reply:
Sure, done